### DISCUSSION TITLE
Project name poll

### DISCUSSION THREAD

**Discussion by earthsound** [input-leap/input-leap#1470] [Polls]
As @shymega [mentioned](https://github.com/input-leap/input-leap/pull/1438#issuecomment-1032870529) a name is needed for this fork. 

I like his suggestion of using 'Input Leap' for the application name, `INPUTLEAP` or `inputleap` for CMake vars and C++ codebase, and Input-Leap for the Git stuff.

**Comment by shymega** [comment]
I'm not keen on the name honestly. Prefer either 'Dash' or 'Leapto'. I can settle with my original suggestion, but I feel that it's a bit convolutd.

**Reply by earthsound** [reply]
Since there are only 2 votes, I can add those to the poll, if you'd like (adding/changing is destructive and resets votes). 
The only thing I'd add is that Dash is used by other apps. Even Leapto is used elsewhere. E.g.:
[Dash](https://kapeli.com/dash)
[Leapto](https://www.leapto.work/)
Let me know if there are other names you'd like and I'll add those, too.

**Reply by shymega** [reply]
I think, yes, it looks like they're taken. I've had a think, and I think we should settle on the Input Leap name, with conventions. The conventions have been what's catching me out. This poll could probably be closed for now.

**Reply by donuts-are-good** [reply]
Name it Mousebringer-Keeper-Of-Pointers and ill donate 100 bucks. 

(sincere offer, USD)

**Comment by OptimusGREEN** [comment]
Most screens are glass, therefor GlassHopper

**Reply by shymega** [reply]
We've already decided, and renaming has finished.

**Comment by shymega** [comment]
To be clear, we've almost finished, if not already, renaming.

**Reply by OptimusGREEN** [reply]
Oh man, I thought I'd won the internet with that suggestion as well.
